iap_watcher/iap_watcher.py

```diff
@@ -40,8 +40,8 @@ def _RetryIfValueIsEmptyHandler(value, output_file):
     logging.info('The latest value is %s.', value)
     metadata = json.loads(value)
     if metadata['enabled']:
-      f = open(output_file, 'w')
-      f.close()
+      with open(output_file, 'w') as file:
```

Comment:
actually I guess you could do

with open(output_file, 'w'):

without naming it